README.md

```diff
@@ -2,8 +2,6 @@
 Let's collect the data of Woman on GitHub, who inspired you to code more, learn more.
 
 **Note:** Please maintain the below list in alphabetical order.
-
-[alicetragedy](https://github.com/alicetragedy)
```

Comment:
@thefourtheye  no issues :). On a side note I have not pushed the changes, should I?
or let it stay.